config/dictionaries/resource.json

```diff
@@ -288,22 +313,43 @@
       {
         "id": "move-device",
         "summary": "Move device to another application",
+        "description": "",
         "method": "PATCH",
         "endpoint": "/v5/device(<ID>)",
         "filters": "",
         "data": "{\n    \"belongs_to__application\": \"<APP ID>\"\n}"
-      },
+	  },
+	  {
+        "id": "get-target-release",
+        "summary": "Get the release device is pinned to",
+        "description": "Note: should_be_running__release will output unset or null if the device isn't pinned to any release. For an unpinned device, the device.is_on__commit field may be used to obtain the device's current state, while the application.commit field represents the device's target state.",
```

Comment:
NIT: _"will output"_ sounds a bit odd to me. How about _"will be"_ instead?  
(An application may print some output to the console when it is executed, but a field by itself does not output anything...) :-)

That's focusing on the writing only, not on whether it is true that `is_on__commit` represents the current state while `application.commit` represents the target state.

Reply:
As for the latter part, @pdcastro that is accurate. Unpinned devices have unset/null `should_be_running__release` and will target to `application.commit`

Reply:
Thanks, resolving!